### PR TITLE
ROC-7281 Add messages to exceptions

### PR DESCRIPTION
**Pull request by andrewwa-kainos**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-7281

### Change description ###
Find constructor method references for exceptions and give them messages

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimantResponseContentProvider.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5008797&bid=16522281)

File: src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProviderTest.java [hunk 1]
```diff
@@ -82,8 +82,10 @@ public void shouldProvideClaimantNameWithTradingAsWhenSoleTrader() {
 
         Map<String, Object> content = provider.createContent(claim, DEFENDANT_PIN);
 
-        String expected = String.format("%s T/A %s", claimant.getName(),
-            claimant.getBusinessName().orElseThrow(IllegalStateException::new));
+        String expected = String.format("%s T/A %s",
+            claimant.getName(),
+            claimant.getBusinessName()
```
Comment (Valkrys) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProviderTest.java]: Could the error not also be the missing claimant name?

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimantResponseContentProvider.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5008797&bid=16522281)

**Comment by akritis22** [comment]
Looks good to me. Just a suggestion - all the error messages can be pulled into a utils file and can be used from there.

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimantResponseContentProvider.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5008797&bid=16522281)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/uk/gov/hmcts/cmc/claimstore/utils/CommonErrors.java  1
         

Complexity decreasing per file
==============================
+ src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimantResponseContentProvider.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5008797&bid=16522281)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/uk/gov/hmcts/cmc/claimstore/utils/CommonErrors.java  1
         

Complexity decreasing per file
==============================
+ src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimantResponseContentProvider.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5008797&bid=16522281)

**Pull request merged**